x-pack/plugins/ingest_pipelines/server/routes/api/create.ts

```diff
@@ -34,7 +34,7 @@ export const registerCreateRoute = ({
       const pipeline = req.body as Pipeline;
 
       // eslint-disable-next-line @typescript-eslint/naming-convention
-      const { name, description, processors, version, on_failure } = pipeline;
+      const { name, description, processors, version, on_failure, _meta } = pipeline;
```

Comment:
are there any server side tests that we need to update for making sure the meta field is also being sent out to es?

Reply:
Yes, thanks for letting me know about the server side tests. I updated them in the next commit.